README.adoc

```diff
@@ -355,11 +319,12 @@ You see the same JSON file that you saw previously.
 
 To stop your container, run the `docker stop rest-app` command from the command line.
 
-If a problem occurs and your container exits prematurely, the container does not appear in the container
-list that the `docker ps` command displays. Instead, your container appears with an `Exited`
-status when you run the `docker ps -a` command. Run the `docker rm rest-app` command to remove your container.
-Next, check your Dockerfile for potential problems. You can also run the container in the foreground to
-see whether any error messages are displayed.
+If a problem occurs and your container exits prematurely, the container will not appear in the container
+list that the `docker ps` command displays. Instead, your container will appear with an `Exited`
```

Comment:
"...container **appears** with an..."

Reply:
also a 1st type conditional?

Reply:
Because the actual condition is not stated in this sentence, I think we'll be OK with using the present tense with "Instead, your container appears with an..." The condition is attached to the previous sentence, but there is no condition stated in the sentence that begins on line 323.

Reply:
ah ok, that also makes sense. I will change it to present tense as you suggested.